javascript-build-logic/src/main/kotlin/android-base-convention.gradle.kts

```diff
@@ -5,10 +5,15 @@
 import com.android.build.gradle.BaseExtension
 
 configure<BaseExtension> {
-    compileSdkVersion(30)
+    compileSdkVersion(36)
 
     defaultConfig {
-        minSdkVersion(18)
-        targetSdkVersion(30)
+        minSdk = 24
```

Comment:
is it required?

Reply:
minimal new sdk - 21 with actual libs versions, but extentions for versions catalog need 24, i'm can delete extentions and set 21, but not less